src/builder/edit_channel.rs

```diff
@@ -115,4 +119,59 @@ impl EditChannel {
 
         self
     }
+
+    /// A set of overwrites defining what a user or a user carrying a certain role can
+    /// and cannot do.
+    ///
+    /// # Example
+    ///
+    /// Inheriting permissions from an exisiting channel:
+    ///
+    /// ```rust,no_run
+    /// # use serenity::{http::Http, model::id::GuildId};
+    /// # use std::sync::Arc;
+    /// #
+    /// # async fn run() -> Result<(), Box<dyn std::error::Error>> {
+    /// #     let http = Arc::new(Http::default());
+    /// #     let mut guild = GuildId(0).to_partial_guild(&http).await?;
+    /// use serenity::model::channel::{PermissionOverwrite, PermissionOverwriteType};
+    /// use serenity::model::id::UserId;
+    /// use serenity::model::permissions::Permissions;
+    ///
+    /// // Assuming a guild have already been bound.
```

Comment:
"guild" is the subject of the sentence, which is in third person.
```suggestion
    /// // Assuming a guild has already been bound.
```

This typo is also present in the example for [`CreateChannel::permissions`][perms], could you fix it there too?

[perms]: https://github.com/serenity-rs/serenity/blob/current/src/builder/create_channel.rs#L108

Reply:
Fixed both. Considering it was a direct copy of that function I'm not surprised, I noticed the `create_channel` after I submitted and just hoped it was overlooked 😜